### PR TITLE
Update swagger

### PR DESCRIPTION
**Pull request by PowellSoftware**
---
When submitting a connector, please check the following conditions for your PR.

- [ X] I attest that the connector works and I verified by deploying and testing all the operations.
- [ X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ X] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@PowellSoftware Are you sure you want to submit artifacts of the `Powell Teams` connector ?

If yes, Please upload the the files you have provided in the certification portal with update, if any.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@PowellSoftware 

Documentation issues (quick fixes):

Title and Description: The title and description of your connector are incorrect, Please undo the update for these properties and update the info section with below property values:

"title": "Powell Teams",
"description": "Powell Teams API endpoints allows you to manage the governance of your teams (creation / update of your teams)."

Contact - The contact section is also need to be restored like below:

"contact": {
"name": "Powell Teams Support",
"url": "https://support.powell-software.com/",
"email": "support-teams@powellsoftware.zendesk.com"
},

Indentations - The properties are not aligned properly and there is longer tab index between the properties in the swagger and apiProperties.json files.

File: certified-connectors/Powell Teams/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Powell Teams
+
+Powell Teams API endpoints documentation.
```
Comment (sriyen-msft) [inline certified-connectors/Powell Teams/README.md:3]: @PowellSoftware Please provide a valid description of your connector which everyone can understand

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Powell Teams/README.md (hunk 1 above)
Comment (PowellSoftware) [reply certified-connectors/Powell Teams/README.md]: Ok, we just updated the description in the readme file.
=>
Powell Teams API endpoints allows you to manage the governance of your teams (creation / update of your teams).

Comment (PowellSoftware) [reply certified-connectors/Powell Teams/README.md]: We put the initial code which had been validated previously. Can you confirm that this is right for you?
=> apiDefinition.swagger.json

`"swagger": "2.0",
  "info": {
    "title": "Powell.Teams.Web.PublicAPI",
    "description": "Powell Teams API endpoints documentation.",
    "version": "V1.1",
    "contact": {
      "name": "Powell Teams support",
      "url": "https://support.powell-software.com/hc/en-us/requests/new",
      "email": "support-teams@powellsoftware.zendesk.com"
    }`

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**